deploy-apps/canary-deploy.html.md.erb

```diff
@@ -0,0 +1,387 @@
+---
+title: Configuring canary app deployments
+owner: CAPI
+---
+
+You can use Cloud Foundry Command Line Interface (cf CLI) commands
+or the Cloud Foundry API (CAPI) to push your apps to <%= vars.app_runtime_first %> using a canary deployment.
+
+For information about the traditional method for addressing app downtime while pushing app updates,
+see [Using blue-green deployment to reduce downtime and risk](./blue-green.html).
+
+For more information about CAPI, see the
+[Cloud Foundry API (CAPI) documentation](https://v3-apidocs.cloudfoundry.org/).
+
+Canary deployments will allow the users to deploy a new version of an application and check if it is running as expected,
+afterwards continue the rolling deployment of the rest of the instances.
+
+The expected workflow for this feature is:
+1. Deploy a single canary instance with the changed code
+1. The user will manually validate that the instance is running as expected
```

Comment:
suggestion: `Manually verify that canary instance with new code runs as expected`